plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -80,6 +83,22 @@ def _onWriteStarted(self, output_device):
                 Logger.log("d", "'info/send_slice_info' is turned off.")
                 return # Do nothing, user does not want to send data
 
+            # Listing all files placed on the buildplate
+            filenames = []
+            for node in DepthFirstIterator(CuraApplication.getInstance().getController().getScene().getRoot()):
+                if type(node) is not SceneNode or not node.getMeshData():
+                    continue    
+                filenames.append(node.getMeshData().getFileName())
+
+            # Creating md5sums and formatting them as discussed on JIRA
+            modelhashes = [hashlib.md5(open(filename, 'rb').read()).hexdigest() for filename in filenames]            
+            modelhash_formatted = ""
+            if modelhashes:
+                modelhash_formatted = str(modelhashes[0])
+                if len(modelhashes) > 1:
```

Comment:
You don't need to check for length. If there is only one entry, the for loop won't be triggered in the first place.